.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,8 @@
-* Tracking issue: #
+ONE-LINE DESCRIPTION (from TRACKING ISSUE)
+
+* Tracking issue: #NNN
 * [Rendered version](/my-branch/text/xxxx.md)
```

Comment:
nit: I found this link didn't quite work last time I tried. The relative url was broken I think, but I can't recall exactly.

Reply:
Removed it